packages/liferay-npm-scripts/src/utils/spawnSync.js

```diff
@@ -12,9 +12,9 @@ const {spawn,} = require('cross-spawn');
  * @param {Object=} options={} Options to pass to spawn.sync
  */
 module.exports = function(command, args = [], options = {}) {
-	const env = process.env;
-
-	env.PATH = `${PATH}:${env.PATH}`;
+	const env = Object.assign({}, process.env, {
+		PATH: `${PATH}:${process.env.PATH}`,
+	});
```

Comment:
(nit) We're already using spread syntax in this file, so it is probably safe to do this without `Object.assign` like I suggested [here](https://github.com/jbalsas/liferay-npm-tools/commit/6f2ea6cac3208109b29f5db442fd7214eb1273ab#commitcomment-32700044).